arkane/gaussian.py

```diff
@@ -353,9 +353,12 @@ def load_scan_energies(self):
         rigid_scan = False
 
         vlist = []  # The array of potentials at each scan angle
-
+        angle = []  # The corresponding angles for each scan
         # Parse the Gaussian log file, extracting the energies of each
         # optimized conformer in the scan
+        scan_number = 0
+        scan_angle = self._load_scan_angle() * np.pi / 180  # convert to radians
+        failed_scans = 0
```

Comment:
can you do that in one line, `scan_number , failed_scans  = 0, 0`?

Reply:
fixed